src/post/Write/editorReducer.js

```diff
@@ -42,60 +47,77 @@ const editor = (state = defaultState, action) => {
         success: false,
         loadingImg: false,
       };
-    case editorActions.CREATE_POST_START:
+    case editorActions.CREATE_POST.START:
```

Comment:
This and other actions below it (except `UPDATE_LAST_SETTINGS`) won't work now.
You forgot to update this after removing `createAsyncActionType`.